src/fsharp/DependencyManager/NativeDllResolveHandler.fs

```diff
@@ -111,25 +111,26 @@ type NativeDllResolveHandler (nativeProbingRoots: NativeResolutionProbe) =
         else
             None
 
-    let appendSemiColon (p: string) =
-        if not(p.EndsWith(";", StringComparison.OrdinalIgnoreCase)) then
-            p + ";"
+    let appendPathSeparator (p: string) =
+        let separator = string string System.IO.Path.PathSeparator
```

Comment:
```suggestion
        let separator = string System.IO.Path.PathSeparator
```
Double string, just to make sure? :)

Reply:
I have no words.  I'm a professional, everybody!